src/app/src/App.jsx

```diff
@@ -94,7 +95,7 @@ class App extends Component {
         const { fetchingFeatureFlags, embed } = this.props;
 
         const mainPanelStyle = embed
-            ? { ...appStyles.mainPanelStyle, bottom: 0, top: 0 }
+            ? { ...appStyles.mainPanelStyle, top: 0 }
```

Comment:
If the `bottom` rule is overridden in embed mode, the footer overlaps with the app.